tests/conftest.py

```diff
@@ -57,3 +61,15 @@ async def nats_jetstream(
     jetstream: Final = nats.jetstream()
     yield jetstream
     await nats.close()
+
+
+@pytest.fixture()
+async def nats_result_backend(
+    nats_urls: List[str],
+) -> AsyncGenerator[NATSObjectStoreResultBackend[_ReturnType], None]:
+    backend: NATSObjectStoreResultBackend[_ReturnType] = NATSObjectStoreResultBackend(
```

Comment:
Is type annotation here really necessary?

Reply:
AFAIK fixtures are always matched by name, so not necessary from pytest point of view.

I didn't even look at this part, but I usually find the more type hints the better.

Reply:
Okay, its not a deal breaker for me)